src/materials/DamagePlasticityStressUpdate.C

```diff
@@ -0,0 +1,682 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/*                       BlackBear                              */
+/*                                                              */
+/*           (c) 2017 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+#include "DamagePlasticityStressUpdate.h"
+#include "libmesh/utility.h"
+
+registerMooseObject("BlackBearApp", DamagePlasticityStressUpdate);
+
+InputParameters
+DamagePlasticityStressUpdate::validParams()
+{
+  InputParameters params = MultiParameterPlasticityStressUpdate::validParams();
+  params.addParam<Real>(
+      "yield_function_tolerance",
+      "If the yield function is less than this amount, the (stress, internal parameters) are "
+      "deemed admissible.  A std::vector of tolerances must be entered for the multi-surface case");
+  params.addRangeCheckedParam<Real>("factor_relating_biaxial_unixial_cmp_str",
```

Comment:
I'd suggest changing this to `biaxial_uniaxial_compressive_stress_factor` (or strain, if that's what it is).

Reply:
Addressed